rubrik_cdm/rubrik_cdm.py

```diff
@@ -358,7 +358,15 @@ def setup_cluster(self, cluster_name, admin_email, admin_password, management_ga
         bootstrap_config["name"] = cluster_name
         bootstrap_config["dnsNameservers"] = dns_nameservers
         bootstrap_config["dnsSearchDomains"] = dns_search_domains
-        bootstrap_config["ntpServers"] = ntp_servers
+
+        if self.minimum_installed_cdm_version(5.0) is False:
+            bootstrap_config["ntpServers"] = ntp_servers
+        else:
+            bootstrap_config["ntpServerConfigs"] = []
+            for ntp in ntp_servers:
+                tempNtp = {}
```

Comment:
`tempNtp = {}` should be `temp_ntp` in terms of formatting. 

Can we also come up with something more descriptive for the variable name? Ideally someone who has never looked at the code understands what is happening here right off the bat.

Also, we're you able to verify this call works on a non-bootstrapped cluster?

Reply:
Will do. I'll be able to complete a verification test by next week and submit the changes requested.

Reply:
@drew-russell Made the changes you requested and tested them.